### PR TITLE
some small changes for akwam, mycima

### PR DESCRIPTION
**Pull request by ImZaw**
Akwam less requests to get media links
Mycima added recommendations and anime

### CODE REVIEW

File: app/src/main/java/com/lagradost/cloudstream3/movieproviders/MyCimaProvider.kt [hunk 1]
```diff
@@ -13,7 +13,7 @@ class MyCimaProvider : MainAPI() {
     override var name = "MyCima"
     override val usesWebView = false
     override val hasMainPage = true
-    override val supportedTypes = setOf(TvType.TvSeries, TvType.Movie)
+    override val supportedTypes = setOf(TvType.TvSeries, TvType.Movie, TvType.Anime)
```
Comment (LagradOst) [inline app/src/main/java/com/lagradost/cloudstream3/movieproviders/MyCimaProvider.kt:16]: Why 🤨

**Review by LagradOst — APPROVED** [review]
Idk why you include it in anime when it cant even return anime response?

File: app/src/main/java/com/lagradost/cloudstream3/movieproviders/MyCimaProvider.kt (hunk 1 above)
Comment (ImZaw) [reply app/src/main/java/com/lagradost/cloudstream3/movieproviders/MyCimaProvider.kt:16]: wdym, they got anime in the site and i loaded them as series i guess? is this a stupid thing to do?

Comment (LagradOst) [reply app/src/main/java/com/lagradost/cloudstream3/movieproviders/MyCimaProvider.kt:16]: I guess that would work, however they should be marked as anime then

**Pull request merged**